dvc/ignore.py

```diff
@@ -56,7 +75,7 @@ def __call__(self, root, dirs, files):
 
         return dirs, files
 
-    def matches(self, dirname, basename, is_dir=False):
+    def _get_normalize_path(self, dirname, basename):
```

Comment:
NOTE: line 88 `return False`. That does not match `_get..`

Reply:
Any `OutofRepoException` use case.